components/tidb_query/src/rpn_expr/impl_json.rs

```diff
@@ -17,6 +17,75 @@ fn json_type(arg: &Option<Json>) -> Result<Option<Bytes>> {
         .map(|json_arg| Bytes::from(json_arg.json_type())))
 }
 
+#[rpn_fn(raw_varg, min_args=2, extra_validator = json_modify_validator)]
```

Comment:
` = ` instead of `=`

Reply:
> `(space)=(space)` instead of `=`

What do you mean by this? I'm not clear what you mean.

Reply:
`min_args = 2` instead of `min_args=2`